packages/rocketchat-integrations/server/api/api.js

```diff
@@ -0,0 +1,358 @@
+/* globals Api Meteor Restivus logger processWebhookMessage*/
+// TODO: remove globals
+import vm from 'vm';
+import moment from 'moment';
+
+const compiledScripts = {};
+
+const buildSandbox = function(store = {}) {
```

Comment:
It is the "way" that I chose ~wrong~. I'll fix it. There are one difference if I tried to overwrite the function.

```javascript
const buildSandbox = function(store = {}) {}
.
.
.
buildSandbox = anotherFunction(){}<- exception

```